### ISSUE TITLE
Enum value of a request path parameter

### ISSUE THREAD

**Issue by 0x003e** [kdy1/rweb#30]
Hi everyone,

Is it possible to implement Enum type in path ? :

![image](https://user-images.githubusercontent.com/12722481/73456078-8500a980-4371-11ea-85e0-038d2aa5b891.png)

```rust

#[get("/{color}")]
async fn get_color(color: Color) -> Result<impl rweb::Reply, Rejection> {

}

#[derive(Debug, Serialize, Deserialize, Schema)]
pub enum Color {
    BLACK,
    BLUE,
}

impl std::str::FromStr for Color {
    type Err = &'static str;
    fn from_str(s: &str) -> Result<Self, Self::Err> {
        match s {
            "black" => Ok(Color::BLACK),
            "blue" => Ok(Color::BLUE),
            _ => Err("ERR"),
        }
    }
}

impl fmt::Display for Color {
    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
        write!(f, "{:?}", self)
    }
}
```

```
---
openapi: 3.0.1
info:
  title: ""
  version: ""
paths:
  "/{color}":
    get:
      parameters:
        - in: path
          name: color
          required: true
          schema:
            type: string
            enum:
              - black
              - blue
      responses:
        "200":
          description: ""
          content:
            text/plain:
              schema:
                type: string
        
```

**Comment by kdy1** [comment]
I'm sorry, but it's not possible currently.
I forgot implementing doc generator clike enums.

**Comment by kdy1** [comment]
@0x003e I'll implement it soon.

**Comment by kdy1** [comment]
@0x003e I fixed it and published v0.4.1.

You need to attach `#[serde(rename = "black")]`, like

```rust

#[derive(Debug, Serialize, Deserialize, Schema)]
pub enum Color {
    #[serde(rename = "black")]
    BLACK,
    #[serde(rename = "blue")]
    BLUE,
}
```

**Comment by 0x003e** [comment]
Thank you @kdy1
